src/modules/dashboard/components/Extensions/ExtensionDetails.tsx

```diff
@@ -0,0 +1,338 @@
+            <UserAvatar
+              address={installedExtension.details.installedBy}
+              size="xs"
+              notSet={false}
+            />
+            <span className={styles.installedByAddress}>
+              <MaskedAddress address={installedExtension.details.installedBy} />
+            </span>
+          </span>
+        ),
+      },
+      {
+        label: MSG.dateInstalled,
+        value: <FormattedDate value={installedExtension.details.installedAt} />,
+      },
+      {
+        label: MSG.versionInstalled,
+        value: `v${extension.currentVersion}`,
+      },
+      {
+        label: MSG.contractAddress,
+        value: <CopyableAddress>{installedExtension.address}</CopyableAddress>,
+      },
+      {
+        label: MSG.developer,
+        value: 'Colony',
+      },
+    ];
+  } else {
+    tableData = [
+      {
+        label: MSG.status,
+        value: <ExtensionStatus installedExtension={installedExtension} />,
+      },
+      {
+        label: MSG.dateCreated,
+        value: <FormattedDate value={extension.createdAt} />,
+      },
+      {
+        label: MSG.latestVersion,
+        value: `v${extension.currentVersion}`,
+      },
+      {
+        label: MSG.developer,
+        value: 'Colony',
+      },
+    ];
+  }
+  const extensionUrl = `/colony/${colonyName}/extensions/${extensionId}`;
+  const breadCrumbs: Crumb[] = [
+    [MSG.title, `/colony/${colonyName}/extensions`],
+    [extension.name, match.url === extensionUrl ? '' : extensionUrl],
+  ];
+  if (match.path === COLONY_EXTENSION_SETUP_ROUTE) {
+    breadCrumbs.push(MSG.setup);
+  }
+  return (
+    <div className={styles.main}>
+      <BreadCrumb elements={breadCrumbs} />
+      <hr />
+      <div className={styles.content}>
+        <Switch>
+          <Route
+            exact
+            path={COLONY_EXTENSION_DETAILS_ROUTE}
+            component={() => (
+              <div>
+                <Heading
+                  tagName="h3"
+                  appearance={{ size: 'medium', margin: 'small' }}
+                  text={extension.name}
+                />
+                <FormattedMessage {...extension.description} />
+              </div>
+            )}
+          />
+          <Route
+            exact
+            path={COLONY_EXTENSION_SETUP_ROUTE}
+            component={() => {
+              if (
+                !canInstall ||
+                !installedExtension ||
+                (installedExtension.details.initialized &&
+                  !installedExtension.details.missingPermissions.length)
+              ) {
+                return <Redirect to={extensionUrl} />;
+              }
+              return (
+                <ExtensionSetup
+                  extension={extension}
+                  installedExtension={installedExtension}
+                  colonyAddress={colonyAddress}
+                />
+              );
+            }}
+          />
+        </Switch>
+        <aside>
```

Comment:
Was the per-domains install functionality required anymore, at least for this release ?

![Screenshot from 2020-11-17 14-36-14](https://user-images.githubusercontent.com/1193222/99391255-9efc8000-28e2-11eb-8155-93b91a08f150.png)
![Screenshot from 2020-11-17 14-36-53](https://user-images.githubusercontent.com/1193222/99391259-a02dad00-28e2-11eb-818c-7b912c7a6d6b.png)